vue/components/ui/molecules/select/select.vue

```diff
@@ -398,7 +463,11 @@ export const Select = {
                 return;
             }
 
-            this.setValue(this.options[this.highlighted].value);
+            if (!this.filteredOptions[this.highlighted]) {

```

Comment:
Why is this needed?

Reply:
this is for the case in which the user presses enter but there's nothing highlighted in the list considering what's written in the select filter.

I personally found this to be the best way to handle the case - no action.

![image](https://user-images.githubusercontent.com/24736423/119860105-e9079380-bf0d-11eb-8b9e-bcfb012e78e4.png)